### ISSUE TITLE
r1_comp_groups download for 'birds' does not work

### ISSUE THREAD

**Issue by bgerstner90** [ropensci/rredlist#40]
I have been having some issues when trying to download the comprehensive list of species in the 'birds' group. The code I am using works for mammals, but for some reason, I am unable to do this for birds. 

This is the code that I am using in R:

```r
all_mammals <-rl_comp_groups(group = c('mammals'), key = token) # token is my IUCN key. This code works and returns a list of all species in the group 'mammal'.
all_birds <-rl_comp_groups(group = c('birds'), key = token) # This does not work.
```

Is this an issue with the API? I have checked that the group name is correct. Any help would be appreciated.

**Comment by sckott** [comment]
I get the same result. 

We'll have to ask the API maintainer - we'll see if we get a response, we may not

**Comment by sckott** [comment]
The API maintainer responded that they'll look into it.

**Comment by sckott** [comment]
@bgerstner90  looks like this is working again

**Comment by rdornas** [comment]
Hello and sorry to bother. It's not working for me. 'Mammals' is working but 'birds' is not.

Question: what are the choices I can insert in `group` argument?

**Comment by sckott** [comment]
as it says in the documentation for the `group` parameter, if you run `rl_comp_groups()` you should get a list of the groups you can use 

it seems the redlist service is down right now for me so i can't check if mammals is working or not

**Comment by rdornas** [comment]
Ok, Scott, thank you. I was trying to retrieve this information but I was having a 'bad gateway' and I could not find the groups. I was not supposing the service was down but I was no sure, thank you for your information.

Anyway, mammals stopped working so I think I got lucky when I first searched for mammals. At the moment no search is being processed probably because the service is down. 

Thank you!
